app/templates/views/download.html

```diff
@@ -19,9 +19,9 @@ <h1 class="govuk-heading-l">Download your file</h1>
     <p>
       If you have any questions,
       {% if contact_info_type == "link" %}
-        <a href={{ service_contact_info }} class="govuk-link"> contact {{ service_name }}</a>.
+        <a href="{{ service_contact_info }}" class="govuk-link"> contact {{ service_name }}</a>.
       {% elif contact_info_type == "email" %}
-        email <a href={{ "mailto:" + service_contact_info }} class="govuk-link">{{service_contact_info}}</a>.
+        email <a href="{{ "mailto:" + service_contact_info }}" class="govuk-link">{{service_contact_info}}</a>.
```

Comment:
I think in this instance it should be:

`email <a href="mailto:{{ service_contact_info }}" class="govuk-link">{{service_contact_info}}</a>.`

Reply:
Or actually, I think what you've written would work but it seems a slightly awkward way to do it compared to my suggestion (feel free to disagree obviously)